src/transforms/scale.jl

```diff
@@ -35,20 +35,17 @@ Scale(r"[ace]", low=0.3, high=0.7)
 
 * The `low` and `high` values are restricted to the interval [0, 1].
 """
-struct Scale{S<:ColSpec,T<:Real} <: ColwiseFeatureTransform
+struct Scale{S<:ColSpec} <: ColwiseFeatureTransform
   colspec::S
-  low::T
-  high::T
+  low::Float64
+  high::Float64
```

Comment:
I think we can still use a type parameter T here to avoid a strict design. The apply function will take care of converting the type to eltype of x anyways.